docs/dotfiles.md

```diff
@@ -60,3 +60,30 @@ sudo apt update
 # Install some of my favorite tools every time my workspace boots
 sudo apt install -y neovim fish cargo
 ```
+
+## Install or Setup script support
+
+User can have either of the following setup script files in their dotfiles repo:
+
+- install.sh
+- install
+- bootstrap.sh
+- bootstrap
+- script/bootstrap
+- setup.sh
+- setup
+- script/setup
+
+If any of the above files are found in the specified order, Coder will execute the first match, and the rest will be skipped.
+
+The Setup scripts must be executable. If they are not, Coder will not execute them. If the script is not executable, you can make it executable by running:
```

Comment:
```suggestion
The setup script must be executable, otherwise the dotfiles setup will fail. If you encounter this issue, you can fix it by making the script executable using the following commands:
```